CMPUT301Project/app/src/main/java/com/example/GroupProject/EditHabitActivity.java

```diff
@@ -80,7 +80,8 @@ protected void onCreate(Bundle savedInstanceState) {
             Map<String, Object> editHabit = new HashMap<>();
 
             // Add all these to Firestore
-            String habitTitle = etHabitTitle.getText().toString();
+//            String habitTitle = etHabitTitle.getText().toString();
+            String habitTitle = habit.getTitle();
```

Comment:
There should be a way to make the habitTitle field uneditable instead of doing this. Try making the habitTitle field uneditable and keep everything else the same.